sb1-scope.json

```diff
@@ -0,0 +1,167 @@
+{
```

Comment:
Where is this file used?

Reply:
Apologies for the missing context. It's now explained at https://confluence.intern.sparebank1.no/x/ywGDVQ